Makefile

```diff
@@ -390,30 +394,30 @@ $(IELE_LIB)/standalone/iele-testing-kompiled/%: $(BUILD_DIR)/standalone/iele-tes
 	@mkdir -p $(dir $@)
 	$(INSTALL) $(dir $<)$* $@
 
-$(IELE_LIB)/check/well-formedness-kompiled/%: $(BUILD_DIR)/check/well-formedness-kompiled/interpreter
-	install -D $(dir $<)$* $@
-
 $(IELE_LIB)/kore-json.py: $(IELE_DIR)/kore-json.py
 	@mkdir -p $(dir $@)
 	$(INSTALL) $< $@
 
-$(INSTALL_BIN)/%: $(IELE_BIN)/%
-	@mkdir -p $(dir $@)
-	$(INSTALL) $< $@
-
-$(INSTALL_LIB)/%: $(IELE_LIB)/%
-	@mkdir -p $(dir $@)
-	$(INSTALL) $< $@
-
 $(INSTALL_BIN)/iele-interpreter: $(INSTALL_LIB)/standalone/iele-testing-kompiled/syntaxDefinition.kore
 
 $(INSTALL_BIN)/iele-check: $(INSTALL_LIB)/check/well-formedness-kompiled/compiled.bin
 $(INSTALL_BIN)/iele-check: $(INSTALL_LIB)/check/well-formedness-kompiled/interpreter
 
 $(INSTALL_BIN)/kiele: $(INSTALL_LIB)/kore-json.py
 $(INSTALL_BIN)/kiele: $(INSTALL_LIB)/version
+$(INSTALL_BIN)/kiele: $(INSTALL_LIB)/haskell/iele-testing-kompiled/definition.kore
+$(INSTALL_BIN)/kiele: $(INSTALL_LIB)/haskell/iele-testing-kompiled/syntaxDefinition.kore
```

Comment:
@gtrepta The problem is here: the Nix package doesn't include the Haskell backend, but you have made it a prerequisite for installing `kiele`.

Reply:
I'm trying to get these files included in the installation because they are needed by the `kiele interpret --backend haskell` command.

Reply:
@ttuegel So the nix package for K doesn't include the haskell backend? How hard would it be to add that in?

Reply:
The Nix package for K includes the Haskell backend. The Nix package for IELE does not. This should not be harder than pulling it into the package, but I will need you to find a way that those files don't become dependencies of the recipe to install `kiele`, because those will be built at a different time.